test/Unity.TypedFactories.Tests/UnityTypedFactoryTests.cs

```diff
@@ -198,12 +200,7 @@ public void given_instantiated_Sut_when_Create_is_called_with_one_of_the_paramet
 
                 // Act
                 var factory = unityContainer.Resolve<ITest2Factory>();
-                var testClass2 = factory.Create(null, someInstance, string.Empty);
-
-                // Assert
```

Comment:
If we want to leave the `Arrange`/`Act`/`Assert` sections clearly separated, we could rename `testClass2` to something like `factoryFunc` and assign `() => factory.Create(null, someInstance, string.Empty)` to it.

Reply:
I made the requested changes.